streaming/src/test/mock_actor.cc

```diff
@@ -639,10 +639,11 @@ class StreamingWorker {
 }  // namespace ray
 
 int main(int argc, char **argv) {
-  RAY_CHECK(argc == 4);
+  RAY_CHECK(argc == 5);
   auto store_socket = std::string(argv[1]);
   auto raylet_socket = std::string(argv[2]);
   auto node_manager_port = std::stoi(std::string(argv[3]));
+  // auto runtime_env_hash = std::string(argv[4]); // Unused in this test
```

Comment:
Remove it?

Reply:
Above this is a line that RAY_CHECKs the number of arguments, which changed from 4 to 5 in this PR because `--runtime-env-hash` is appended to the command string.  I thought I would add this comment so people know what the fifth argument is, because they might wonder about it.  But if it's not useful I'll remove it